### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by polymeris**
Cedar stack needs to be specified, apparently. Else, cedar-14 will be used.

### CODE REVIEW

**Comment by leafo** [comment]
It should be able to work with both stacks, which rockspec did you use?

**Comment by polymeris** [comment]
`"https://raw.github.com/leafo/heroku-openresty/master/heroku-openresty-1.2.8.6-1.rockspec"`
Just copied it from this readme, will try with the newer one.

**Comment by polymeris** [comment]
- `heroku-openresty-1.5.11.1-1.rockspec` does not work, either.
- `heroku-openresty-dev-2.rockspec` _does_ work.

Maybe the docs should be updated to point to the later.

**Comment by mathieujobin** [comment]
Hello, I am new to Lua
I am trying to test this along with the heroku buildpack using the example https://github.com/leafo/heroku-lapis-example and nothing really works... maybe it would be best to update the example in a working state. I am trying to fix it myself, but with limited knowledge of luarocks, openresty and lua itself, it is a bit hard.

thanks

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ work.
 Create a new app with the [Lua buildpack](https://github.com/leafo/heroku-buildpack-lua).
 
 ```bash
-$ heroku create --buildpack http://github.com/leafo/heroku-buildpack-lua.git
+$ heroku create --buildpack http://github.com/leafo/heroku-buildpack-lua.git --stack cedar
```
Comment (mathieujobin) [inline README.md:33]: `cedar-14` is cedar, their newest stack....

```
[mathieu@hz550 heroku-lapis-example]$ heroku create --buildpack http://github.com/leafo/heroku-buildpack-lua.git --stack cedar --app lapis-example
Creating lapis-example... done, stack is cedar-14
Buildpack set. Next release on lapis-example will use http://github.com/leafo/heroku-buildpack-lua.git.
https://lapis-example.herokuapp.com/ | https://git.heroku.com/lapis-example.git
[mathieu@hz550 heroku-lapis-example]$ heroku stack
=== heroku-lapis-example2 Available Stacks
* cedar-14
[mathieu@hz550 heroku-lapis-example]$ 
```

**Pull request closed without merging**